### PR TITLE
increasing highpass efficiency

### PR DESCRIPTION
**Pull request by iAdityaEmpire**
## Related Issue
Fixes N/A

## Summary
- [x] I have read CONTRIBUTING.md to understand how to contribute to this repository :)

<Please summarize what you are trying to achieve, what changes you made, and how they acheive the desired result.>
Speeding up highpassfilter by taking advantage of existing dependencies

Previous runtime: 1.686479s
New runtime: 0.045012s

## Unit Tests
If your changes touch the `audio` module, please run all of the `audio` tests and paste the output here. Likewise for `image`, `text`, & `video`. If your changes could affect behavior in multiple modules, please run the tests for all potentially affected modules. If you are unsure of which modules might be affected by your changes, please just run all the unit tests.

### Audio
```bash
python -m unittest discover -s augly/tests/audio_tests/ -p "*"
......./home/adityaprasad/.local/lib/python3.8/site-packages/librosa/util/utils.py:2099: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this w
arning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  np.dtype(np.float): np.complex,
/home/adityaprasad/.local/lib/python3.8/site-packages/librosa/util/utils.py:2099: DeprecationWarning: `np.complex` is a deprecated alias for the builtin `complex`. To silence this warn
ing, use `complex` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.complex128` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  np.dtype(np.float): np.complex,
/home/adityaprasad/.local/lib/python3.8/site-packages/librosa/util/utils.py:869: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning,
use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  mag = np.abs(S).astype(np.float)
........../home/adityaprasad/.local/lib/python3.8/site-packages/librosa/core/spectrum.py:1223: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence
this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  time_steps = np.arange(0, D.shape[1], rate, dtype=np.float)
/home/adityaprasad/.local/lib/python3.8/site-packages/librosa/util/utils.py:2099: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning,
 use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  np.dtype(np.float): np.complex,
/home/adityaprasad/.local/lib/python3.8/site-packages/librosa/util/utils.py:2099: DeprecationWarning: `np.complex` is a deprecated alias for the builtin `complex`. To silence this warn
ing, use `complex` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.complex128` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  np.dtype(np.float): np.complex,
/home/adityaprasad/.local/lib/python3.8/site-packages/librosa/core/spectrum.py:1223: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warni
ng, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  time_steps = np.arange(0, D.shape[1], rate, dtype=np.float)
/home/adityaprasad/.local/lib/python3.8/site-packages/librosa/util/utils.py:869: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning,
use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  mag = np.abs(S).astype(np.float)
..................................................
----------------------------------------------------------------------
Ran 67 tests in 5.665s

OK

```

If applicable, test your changes and paste the output here. For example, if your changes affect the requirements/installation, then test installing augly in a fresh conda env, then make sure you are able to import augly & run the unit test

### CODE REVIEW

**Comment by iAdityaEmpire** [comment]
What does `alpha` signify in this filter?

File: augly/audio/functional.py [hunk 1]
```diff
@@ -457,36 +457,30 @@ def high_pass_filter(
     rc = 1 / (2 * math.pi * cutoff_hz)
     dt = 1 / sample_rate
     alpha = rc / (rc + dt)
```
Comment (zpapakipos) [inline augly/audio/functional.py:459]:
```suggestion
```

Looks like these vars aren't used/needed anymore

File: augly/audio/functional.py [hunk 2]
```diff
@@ -457,36 +457,30 @@ def high_pass_filter(
...
+        aug_audio = aug_audio.reshape((aug_audio.shape[-1],))
 
     if metadata is not None:
         audutils.get_metadata(
             metadata=metadata,
             function_name="high_pass_filter",
             dst_audio=high_pass_array,
-            dst_sample_rate=sample_rate,
+            dst_sample_rate=out_sample_rate,
             alpha=alpha,
```
Comment (zpapakipos) [inline augly/audio/functional.py:478]:
```suggestion
```

**Review by zpapakipos — CHANGES_REQUESTED** [review]

**Review by zpapakipos — APPROVED** [review]

File: augly/audio/functional.py [hunk 3]
```diff
@@ -454,39 +454,28 @@ def high_pass_filter(
...
-            )
+    aug_audio, out_sample_rate = sox_effects.apply_effects_tensor(
+        torch.Tensor(audio), sample_rate, [["highpass", str(cutoff_hz)]]
+    )
 
-    if num_channels == 1:
-        high_pass_array = high_pass_array.reshape((high_pass_array.shape[1],))
+    high_pass_array = aug_audio.numpy()
+    if audio.shape[0] == 1:
+        aug_audio = aug_audio.reshape((aug_audio.shape[-1],))
```
Comment (zpapakipos) [inline augly/audio/functional.py:466]:
```suggestion
        high_pass_array = high_pass_array.reshape((aug_audio.shape[-1],))
```
+1 to bug found in https://github.com/facebookresearch/AugLy/pull/153#pullrequestreview-823013558

**Review by zpapakipos — CHANGES_REQUESTED** [review]

File: augly/audio/functional.py (hunk 3 above)
Comment (iAdityaEmpire) [reply augly/audio/functional.py:466]: I should also be calling `high_pass_array` in the .shape(), right?

**Pull request closed without merging**